backend/core/src/seeds/listings.ts

```diff
@@ -84,303 +91,2639 @@ export async function seedListing(app: INestApplicationContext, seed: ListingSee
+  status: ListingStatus.active,
+  waitlistCurrentSize: 0,
+  waitlistMaxSize: 3000,
+  whatToExpect: null,
+}
+
+// Seeds
```

Comment:
Most everything above this is sample data. Below is where I build up the different seeds.